### DISCUSSION TITLE
locales supported?

### DISCUSSION THREAD

**Discussion by metalllus** [flutter-stripe/flutter_stripe#114] [General]
Is it possible to use non-english locale? In my case, I would need Slovak language...

**Comment by remonh87** [comment]
. Based on this [doc]( https://support.stripe.com/questions/supported-languages-for-stripe-checkout ) checkout is localized. You can double check it by running the example app tap on paymentsheet  if Slovak is there. I checked for Swedish and the payment sheet is localized.

**Comment by thorsten-stripe** [comment]
The payment sheet is automatically localized based on device language. See https://github.com/stripe/stripe-react-native/issues/243#issuecomment-849701446

**Comment by angelorobo** [comment]
@thorsten-stripe Is it planned to support an override for the language? If yes, is there an estimation of when this will become available? In my case, the app I'm developing should display in a different language than my device's language.

**Comment by angelorobo** [comment]
Is there a solution for this?

**Comment by sagar-techindustan** [comment]
<key>CFBundleAllowMixedLocalizations</key>
	<string>true</string>
	<key>CFBundleLocalizations</key>
	<array>
		<string>Svenska</string>
		<string>en</string>
	</array>
	
	
	
	just write your language name .
	Note: paste the code in info.plist of your iOS  app folder. Language name should be correct.
 
	
	Stripe payment sheet automatically do translation   as per your language.

**Comment by fulstadev** [comment]
Hi, so there is still no way to provide a concrete locale to the payment sheet, for every case where the language of your app is controlled on the app-level, and not on the device-level? We assumed that's possible, as [the docs](https://pub.dev/packages/flutter_stripe#card-payments) mention that the payment sheet includes localization; does that really only refer to the default device locale..? The `Info.plist` file solution only works with languages with which the app has been initiated. When you change the app language (in-app, e.g. using sth like [this](https://github.com/flutter/flutter/issues/143729#issuecomment-2117436698)), the locale of the payment sheet is still the one with which you initiated the app.

**Reply by remonh87** [reply]
@fulstadev 

like @thorsten-stripe  said it is automatically localised but there is an open ticket for it on stripe:  https://github.com/stripe/stripe-react-native/issues/243#issuecomment-849701446

**Comment by Bujupah** [comment]
Any updates in this area?
